app/styles/pages/events/sessions.scss

```diff
@@ -0,0 +1,17 @@
+.left-float{
```

Comment:
You can use `left floated` or `right floated` for columns. You need to use proper grid structure.